### DISCUSSION TITLE
[Question] How to provide the canvaskit.js locally

### DISCUSSION THREAD

**Discussion by iigx** [flet-dev/flet#2995] [Q&A]
### Question

I am creating a web app that runs entirely on a local area network, but I find that Flets download some common components from the external network, making the entire page load very slow, such as:
 - https://www.gstatic.com/flutter-canvaskit/2e4ba9c6fb499ccd4e81420543783cc7267ae406/chromium/canvaskit.js
 - https://fonts.gstatic.com/s/roboto/v20/KFOmCnqEu92Fr1Me5WZLCzYlKw.ttf
 - https://www.gstatic.com/flutter-canvaskit/2e4ba9c6fb499ccd4e81420543783cc7267ae406/chromium/canvaskit.wasm
How can I avoid it?
![image](https://github.com/flet-dev/flet/assets/46487494/e45bb4cd-e46b-424a-9417-810286580f6a)

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by esimorre** [comment]
you may find ideas in the main.dart.js file, it seems that alt urls can be defined somewhere in flutter
